Sources/Views/PageLayoutViewV2.swift

```diff
@@ -177,6 +177,7 @@ struct NavBar: HTML {
     .inlineStyle("width", "100%")
     .inlineStyle("position", "sticky", media: .mobile)
     .inlineStyle("top", "0", media: .mobile)
+    .inlineStyle("z-index", "9999")
```

Comment:
the video for clips (and episodes) is absolutely positioned and so we need this to make sure the nav renders over it.